lib/cypress-rails/finds_bin.rb

```diff
@@ -0,0 +1,11 @@
+module CypressRails
+  class FindsBin
+    def call(_dir = Dir.pwd)
+      if ENV["KNAPSACK_PRO_CI_NODE_TOTAL"].present? && ENV["KNAPSACK_PRO_CI_NODE_INDEX"].present?
```

Comment:
because we're setting these two ENV vars in travis, and guarding based on them, no changes to local cypress behavior is expected.